stratis_cli_cert.py

```diff
@@ -133,6 +133,18 @@ def _unittest_command(
             self.assertNotEqual(stdout, "")
 
 
+class StratisCliVersionCertify(StratisCertify):
+    """
+    Tests that check that the stratis command returns a version.
+    """
+
+    def test_access_stratis_man_page(self):
```

Comment:
Better change name of test and test header documentation, too.

Reply:
Resolved via force-push.